examples/src/vdf/regular/mod.rs

```diff
@@ -3,10 +3,15 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
-use crate::{Example, ExampleOptions};
+use crate::{
+    fibonacci::{Blake3_192, Blake3_256, Sha3_256},
```

Comment:
Similar to the last comment: it feels odd that we define these in fib examples and them import them from there into other places. We should probably put them into a more central place.